apps/nextjs-website/src/app/page.tsx

```diff
@@ -6,23 +6,16 @@ import News from '@/components/organisms/News/News';
 import ProductsShowcase from '@/components/organisms/ProductsShowcase/ProductsShowcase';
 import { getProducts } from '@/lib/api';
 import { Metadata } from 'next';
+import { makeMetadata } from '@/helpers/metadata.helpers';
 
 export async function generateMetadata(): Promise<Metadata> {
   const { homepage, shared } = translations;
 
-  return {
+  return makeMetadata({
     title: shared.siteTitle,
     description: homepage.productsShowcaseTitle,
-    openGraph: {
-      title: shared.siteTitle,
-      description: homepage.productsShowcaseTitle,
-      url: '/',
-    },
-    twitter: {
-      card: 'summary_large_image',
-      site: shared.siteTitle,
-    },
-  };
+    url: '/',
+  });
```

Comment:
The last time I saw this PR, on the homepage I had the `summary_large_image` as `twitter:card` metatag. Now I'm having that `twitter:card` is equals to `summary`.
Why is this changed?

Reply:
I changed it because we don't really have a `large_image` also for the homepage. So the `summary` would still be a better option